notebooks/official/ray_on_vertex_ai/spark_on_ray_on_vertex_ai.ipynb

```diff
@@ -0,0 +1,1353 @@
+        "id": "ce0ecd10af62"
+      },
+      "outputs": [],
+      "source": [
+        "!docker push {LOCATION}-docker.pkg.dev/{PROJECT_ID}/{DOCKER_REPOSITORY}/{IMAGE_NAME}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "57ad7b911d1d"
+      },
+      "source": [
+        "## Create Ray cluster on Vertex AI\n",
+        "Use the custom container image to create Ray cluster on Vertex AI using Vertex AI Python SDK."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "d48152709457"
+      },
+      "outputs": [],
+      "source": [
+        "CLUSTER_NAME = \"my-rov-cluster\"\n",
+        "CUSTOM_CONTAINER_IMAGE_URI = (\n",
+        "    f\"{LOCATION}-docker.pkg.dev/{PROJECT_ID}/{DOCKER_REPOSITORY}/{IMAGE_NAME}\"\n",
+        ")\n",
+        "CLUSTER_RESOURCE_NAME = (\n",
+        "    f\"projects/{PROJECT_NUMBER}/locations/{LOCATION}/persistentResources/{CLUSTER_NAME}\"\n",
+        ")\n",
+        "RAY_ADDRESS = f\"vertex_ray://{CLUSTER_RESOURCE_NAME}\""
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "7a1451756ff8"
+      },
+      "outputs": [],
+      "source": [
+        "from google.cloud.aiplatform import vertex_ray\n",
+        "from vertex_ray import Resources"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "11778fbceba4"
+      },
+      "outputs": [],
+      "source": [
+        "head_node_type = Resources(\n",
+        "    machine_type=\"n1-standard-16\",\n",
+        "    node_count=1,\n",
+        "    custom_image=CUSTOM_CONTAINER_IMAGE_URI,\n",
+        ")\n",
+        "\n",
+        "worker_node_types = [\n",
+        "    Resources(\n",
+        "        machine_type=\"n1-standard-8\",\n",
+        "        node_count=2,\n",
+        "        custom_image=CUSTOM_CONTAINER_IMAGE_URI,\n",
+        "    )\n",
+        "]\n",
+        "\n",
+        "ray_cluster_resource_name = vertex_ray.create_ray_cluster(\n",
+        "    head_node_type=head_node_type,\n",
+        "    worker_node_types=worker_node_types,\n",
+        "    cluster_name=CLUSTER_NAME,\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "de35e5d0026b"
+      },
+      "source": [
+        "## Spark on Ray on Vertex AI using Ray client\n",
+        "Ray [Task](https://docs.ray.io/en/latest/ray-core/tasks.html#ray-remote-functions) or [Actor](https://docs.ray.io/en/latest/ray-core/actors.html), is required for creating a Spark session with [Ray client](https://docs.ray.io/en/latest/cluster/running-applications/job-submission/ray-client.html) on the Ray on Vertex AI.  The following code shows how a Ray Actor can be used for creating a Spark Session, running a Spark application and stopping a Spark session on a Ray on Vertex AI using RayDP.\n"
```

Comment:
- add comma after "Spark application":
    - "running a Spark application, and stopping a Spark session"